lib/grpnames.g

```diff
@@ -78,62 +78,62 @@ NAMES_OF_SMALL_GROUPS :=
-      "(C5 : C8) : C2", "C4 x (C5 : C4)", "C20 : C4", "C2 x (C5 : C8)",
-      "(C5 : C8) : C2", "(C2 x (C5 : C4)) : C2", "C2 x (C5 : Q8)",
-      "C2 x C4 x D10", "C2 x D40", "(C20 x C2) : C2", "D8 x D10",
-      "(C4 x D10) : C2", "Q8 x D10", "(C4 x D10) : C2",
-      "C2 x C2 x (C5 : C4)", "C2 x ((C10 x C2) : C2)", "C20 x C2 x C2",
-      "C10 x D8", "C10 x Q8", "C5 x ((C4 x C2) : C2)",
+  [ "C13 : C6", "C2 x (C13 : C3)", "C13 x S3", "C3 x D26", "D78", "C78" ],
+  [ "C79" ],
+  [ "C5 : C16", "C80", "C5 : C16", "C8 x D10", "C5 : (C8 : C2)", "C5 : QD16",
+      "D80", "C5 : Q16", "C2 x (C5 : C8)", "C5 : (C8 : C2)", "C4 x (C5 : C4)",
+      "C5 : (C4 : C4)", "C5 : (C4 : C4)", "C5 : ((C4 x C2) : C2)",
+      "C5 : D16", "C5 : QD16", "C5 : QD16", "C5 : Q16",
+      "C5 : ((C4 x C2) : C2)", "C20 x C4", "C5 x ((C4 x C2) : C2)",
+      "C5 x (C4 : C4)", "C40 x C2", "C5 x (C8 : C2)", "C5 x D16",
+      "C5 x QD16", "C5 x Q16", "C5 : (C8 x C2)", "C5 : (C8 : C2)",
+      "C4 x (C5 : C4)", "C5 : (C4 : C4)", "C2 x (C5 : C8)", "C5 : (C8 : C2)",
+      "C5 : ((C4 x C2) : C2)", "C2 x (C5 : Q8)", "C2 x C4 x D10", "C2 x D40",
+      "C5 : (D8 : C2)", "D8 x D10", "C5 : (D8 : C2)", "Q8 x D10",
+      "C5 : (D8 : C2)", "C2 x C2 x (C5 : C4)", "C2 x (C5 : D8)",
+      "C20 x C2 x C2", "C10 x D8", "C10 x Q8", "C5 x ((C4 x C2) : C2)",
       "(C2 x C2 x C2 x C2) : C5", "C2 x C2 x (C5 : C4)", "C2 x C2 x C2 x D10",
       "C10 x C2 x C2 x C2" ],
   [ "C81", "C9 x C9", "(C9 x C3) : C3", "C9 : C9", "C27 x C3", "C27 : C3",
       "(C3 x C3 x C3) : C3", "(C9 x C3) : C3", "(C9 x C3) : C3",
       "C3 . ((C3 x C3) : C3) = (C3 x C3) . (C3 x C3)", "C9 x C3 x C3",
       "C3 x ((C3 x C3) : C3)", "C3 x (C9 : C3)", "(C9 x C3) : C3",
       "C3 x C3 x C3 x C3" ], [ "D82", "C82" ], [ "C83" ],
-  [ "(C7 : C4) : C3", "C4 x (C7 : C3)", "C7 x (C3 : C4)", "C3 x (C7 : C4)",
-      "C21 : C4", "C84", "C2 x ((C7 : C3) : C2)", "S3 x D14",
-      "C2 x C2 x (C7 : C3)", "C7 x A4", "(C14 x C2) : C3", "C6 x D14",
+  [ "C7 : C12", "C4 x (C7 : C3)", "C7 x (C3 : C4)", "C3 x (C7 : C4)",
+      "C3 : (C7 : C4)", "C84", "C2 x (C7 : C6)", "S3 x D14",
+      "C2 x C2 x (C7 : C3)", "C7 x A4", "(C2 x C2) : (C7 : C3)", "C6 x D14",
       "C14 x S3", "D84", "C42 x C2" ], [ "C85" ], [ "D86", "C86" ],
   [ "C87" ],
   [ "C11 : C8", "C88", "C11 : Q8", "C4 x D22", "D88", "C2 x (C11 : C4)",
-      "(C22 x C2) : C2", "C44 x C2", "C11 x D8", "C11 x Q8", "C2 x C2 x D22",
+      "C11 : D8", "C44 x C2", "C11 x D8", "C11 x Q8", "C2 x C2 x D22",
       "C22 x C2 x C2" ], [ "C89" ],
   [ "C5 x D18", "C9 x D10", "D90", "C90", "C3 x C3 x D10", "C15 x S3",
-      "C3 x D30", "C5 x ((C3 x C3) : C2)", "(C15 x C3) : C2", "C30 x C3" ],
+      "C3 x D30", "C5 x ((C3 x C3) : C2)", "(C3 x C3) : D10", "C30 x C3" ],
   [ "C91" ], [ "C23 : C4", "C92", "D92", "C46 x C2" ], [ "C31 : C3", "C93" ],
   [ "D94", "C94" ], [ "C95" ],, [ "C97" ],
   [ "D98", "C98", "C7 x D14", "(C7 x C7) : C2", "C14 x C7" ],
```

Comment:
Many of these changes look like improvements to me; some perhaps are slightly worse -- but I think this is mostly subjective. In any case, I didn't notice anything that struck me as "bad", so I am fine with that part of the change.